string sendClass
int32 count